clean-abap/CleanABAP.md

```diff
@@ -459,6 +460,22 @@ for example prefer `detection_object_types` to something cryptic like `dobjt`.
 
 > Read more in _Chapter 2: Meaningful Names: Use Pronounceable Names_ of [Robert C. Martin's _Clean Code_]
 
+### Use snake_case
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Names](#names) > [This section](#use-snake_case)
+
+ABAP is case insensitive and limits the number of available letters for naming all kind of DDIC-objects
+and variables. For example the maximum length of a method name is 30 characters. The defacto naming convention
+in ABAP is `snake_case`, because ABAP is not case sensitive and camelCase is overwritten when the Pretty Printer / Formatter is activated and not all team members use the 'Keep Camel Case Identifiers' setting in the ADT ABAP Formatter. Also `camelCase` doesn't
+affect DDIC-objects.
+We recommend using `snake_case` consistently instead of `flatcase` or `UPPERCASE`. When you reach the maximum 
+length of an object, try to conscientiously use abbreviations (see [Use same abbreviations everywhere](#use-same-abbreviations-everywhere)).
+
+```ABAP
+" a variable which contains the reponse time measured in milliseconds
+response_time_in_millisec  " ok
+responsetimeinmilliseconds " bad practice
```

Comment:
Can you think of a better example? `response_time_in_milliseconds` still only has 29 characters, so it doesn't really get the point across.

Reply:
maybe like this?
max_response_time_in_millisec
maxresponsetimeinmilliseconds

Reply:
LGTM 👍